network-store-iidm-tck/src/test/java/com/powsybl/network/store/tck/MergeNetworkIT.java

```diff
@@ -205,4 +198,15 @@ public void testMergeAndDetachWithProperties() {
         // FIXME
     }
 
+    @Test
+    @Override
+    public void failMergeWithCommonAreaConflict() {
+        // FIXME
+    }
+
+    @Test
+    @Override
+    public void testMergeAndDetachWithDistinctAreas() {
+        // FIXME
+    }
```

Comment:
We should not override new tests like that, are you sure ?

Reply:
Literally all tests from parent are "disable" in this class.